### ISSUE TITLE
Accessibility: Cannot reorder links using keyboard

### ISSUE THREAD

**Issue by GuySartorelli** [silverstripe/silverstripe-linkfield#409]
This may be specific to linkfield in an elemental block.

When pressing `spacebar` to start reordering links, the inline-editable elemental block closes instead of letting me reorder the links.

Note since this is a bug for existing functionality, it should aim to be fixed in CMS 5.4 if present there.

**Comment by emteknetnz** [comment]
> Note since this is a bug for existing functionality, it should aim to be fixed in CMS 5.4 if present there.

IMO this is a functionality change since it changes the behavior of what the `space` key does from opening to activating a 'move mode'

This is also similar to how you cannot reorder links using keyboard in sitetree. In both cases the functionality simply doesn't exist to reorder with keyboard, only drag and drop. Therefore I'll target this work at `6` not `5.4` since it's adding new functionality

**Comment by GuySartorelli** [comment]
When we implemented linkfield, iirc reordering links with keyboard was part of the functionality. It has just broken since then.

**Comment by emteknetnz** [comment]
This actually already works in 5.4 and 6.1 - there's a focusable drag handle enter/space will activate it, then use the up/down arrow keys to move. There's an existing behat test for this https://github.com/silverstripe/silverstripe-linkfield/pull/226/files#diff-3e4b017fbb5d991ec3e69c1d62682037e86309c42bb91e7bbff696572c33bd32R116
